src/screens/account/account.screen.tsx

```diff
@@ -1,53 +1,93 @@
-import { useCallback, useEffect, useState } from 'react';
-import { observer } from 'mobx-react-lite';
-import { Card, Text } from '../../components/primitive';
-import { accountService } from '../../services';
+import { StatsCard, Text, Button, Avatar, ToggleSwitch } from 'components/primitive';
+import {
+  AccountContainer,
+  AccountInfoContainer,
+  AccountInfo,
+  ProfileContainer,
+  Profile,
+  ProfileInfo,
+  StatsCardContainer,
+  AccountSettingsContainer,
+  StyledDiv,
+  DefaultSettingsContainer,
+  Card,
+  StyledInput,
+  InputContainer,
+  SwitchContainer,
+} from './account.screen.styles';
+// Todo- remove this after integration
 import { stores } from '../../store';
-import { AccountScreenContainer, Title } from './account.screen.styles';
+import Demo from '../../../src/assets/images/demo.png';
```

Comment:
Image imports should go inside image.ts. src prop must be used only for dynamic images.